### PR TITLE
Merge branch 'path' to 'master'

### PR DESCRIPTION
**Pull request by darrenklein**
This merge would incorporate a new feature to the inventory app - the use of a query parameter ('cat') which responds to changes in the UI, or can be used to set the UI by navigating directly to a valid query param.

For example, a user would be able to navigate directly to `http://db.bushwickfoodcoop.org/tools/inventory.html?cat=breads` to view breads; clicking through the list of categories will update the query param to match. Additionally, if a user has selected a category, then begins a search, and then deletes all of the search characters, the UI will return to the state reflected in the URL.

Additional changes:
- The JavaScript has been moved out of the body of the HTML and into an adjacent 'inventory.js' file.
- The JavaScript has been linted to ESLint's Airbnb standards.
- The PHP files have been updated to prefer using single quotes, to avoid unintended string interpolation - http://www.scriptingok.com/tutorial/Single-quotes-vs-double-quotes-in-PHP

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -44,7 +44,7 @@ body {
 .header input {
 	display: inline-block !important;
```
Comment (verakern) [inline main.css]: The indent seems out of place (old and new file), maybe it's a mix of tabs and spaces?

File: inventory.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    }
+  });
+
+  // Update the headline with the current search term.
+  $('itemrow.headline').find('itemname').html(`Search for ${input}`);
+}
+
+// Called when the DOM is ready. Loads all food items from the database.
+function loadFoodItems(queryParam) {
+  $.post('../inventory_get_all_items.php', {}, (data) => {
```
Comment (verakern) [inline inventory.js]: Are the files in different folders now? Maybe I misunderstand why you need the ../

**Review by verakern — COMMENTED** [review]
I love this, thanks for cleaning up so thoroughly! Just two questions

File: inventory.js (hunk 2 above)
Comment (darrenklein) [reply inventory.js]: Ah, that's a typo - initially, I did move the script into a scripts/ directory, but then when I saw how things were arranged on the server, I opted to move the script back out to the parent - oddly, the script still ran ok on my local machine, so I failed to notice. Fixing this now.

File: main.css (hunk 1 above)
Comment (darrenklein) [reply main.css]: Yep, tab and space mix. Fixing now...

File: inventory.js (hunk 2 above)
Comment (darrenklein) [reply inventory.js]: Browser had cached the app with the older file structure, so I didn't notice that it should've broken...

**Pull request merged**